Sources/TuistKit/Commands/GraphCommand.swift

```diff
@@ -81,7 +79,15 @@ public struct GraphCommand: AsyncParsableCommand {
     )
     var outputPath: String?
 
+    public init() {}
+
     public func run() async throws {
+        // If output path is present
+        var absoluteOutputPath: AbsolutePath?
+        if let outputPath {
+            absoluteOutputPath = try AbsolutePath(validating: outputPath, relativeTo: FileHandler.shared.currentPath)
```

Comment:
I'd use [this one](https://github.com/tuist/FileSystem/blob/1bff6d54e90a79706a4a9d77cd081104de430361/Sources/FileSystem/FileSystem.swift#L255) creating an instance of scoped to the command (i.e. a property).

Reply:
You mean like this: https://github.com/tuist/tuist/pull/7363/commits/9a8698aad019755b4489656bcf77660ab8435eaa#diff-b3d112961f66eeb993eb40267f3b8e91015a3e1a9747f240732e4a4291e0c3f2R91 ?